visualization_marker_tutorials/README.md

```diff
@@ -0,0 +1,265 @@
+# visualization_marker_tutorials
+Unlike other displays, the Marker Display lets you visualize data in RViz without RViz knowing anything about interpreting that data.
```

Comment:
It might be better to start with a sentence that emphasizes this is a tutorial about how to render primitive shapes in RVIz. 

It might not be obvious to someone stumbling across this document what a display or the Marker Dispaly is.

Reply:
Good point. I've added an explanation in index.rst.